end-to-end-tests/cypress/e2e/End2EndTests/project-runthrough.cy.js

```diff
@@ -32,9 +32,9 @@ describe("Creating and editing an academy transfer", { tags: '@dev'}, function (
         cy.storeSessionData();
     });
 
-    beforeEach(function () {
-        
+    beforeEach(function () {        
         cy.login();
+        cy.get('[data-test=cookie-banner-accept]').click();
```

Comment:
Looks like we need this after every login() function . Is it worth to add this line of code in login function under command.js ?? @jordan-beard

Reply:
@rdogra-nimble good point. Happy to make this change.

Reply:
Just tried it and it caused tests to fail. Presumably the login method is sometimes being called more than once in a test and when the accept button is not there the second time it fails.

Reply:
No worries. Thanks for trying.